docs/source/manual/core.rst

```diff
@@ -924,6 +924,25 @@ single ``Logger``:
 
 .. _man-core-testing-applications:
 
+Common Logging Configuration
```

Comment:
not really sure where to put common settings in docs, LMK if you'd like me to move this somewhere else.